coding_conventions.md

```diff
@@ -322,3 +326,105 @@ The 'good' example demonstrates how to document the Promise.
  */
 async function go (regionId, billingPeriod) {
 ```
+## Page Title and Active NavBar Logic
+
+When working with pageTitle and activeNavBar, we follow the following conventions to maintain consistency and keep our code clean.
+
+### Page Titles
+
+The `pageTitle` should always be set in the presenter.
```

Comment:
If there is more than one presenter then which one would it go in ?

Reply:
As a convention for the return object variable name should now be called pageData rather than specific to the presenter logic. 

`const pageData = MyPresenter.go()`